fortnox-login.user.js

```diff
@@ -0,0 +1,16 @@
+// ==UserScript==
+// @name        Fortnox login
+// @version     1.0
+// @grant       none
+// @require     https://code.jquery.com/jquery-3.4.1.slim.min.js
+// @include     https://*.fortnox.se/*
+// @downloadURL https://github.com/Wikimedia-Sverige/scripts/raw/master/fortnox-login.user.js
+// ==/UserScript==
+
+$(function() {
```

Comment:
If you use Tampermonkey's default wrapper for this you can remove Jquery. I.e:
```
(function() {
    'use strict';

    // Your code here...
})();
```

Reply:
You've removed the Jquery `require` line, but the wrapper still uses it. This will still work if Jquery is loaded before this runs (e.g. if it's used by Fortnox), but you shouldn't assume that.

Reply:
my bad. I assumed the default wrapper also `require`d it behind the scenes. I'll re-add [since it worked without it I guess Fortnox did load it]

Reply:
That works too, I guess.